thirdparty-api/transaction-patterns-transfer.md

```diff
@@ -0,0 +1,327 @@
+# Transfer API
+
+<!-- vscode-markdown-toc -->
+* [1. Transfers](#Transfers)
+	* [1.1 Discovery](#Discovery)
+	* [1.2 Agreement](#Agreement)
+		* [1.2.1 `POST /thirdpartyRequests/transactions`](#POSTthirdpartyRequeststransactions)
```

Comment:
> [1.2.1 `POST /thirdpartyRequests/transactions`](#POSTthirdpartyRequeststransactions)

Could we use the logical name for this section as well, as all other sections seems to be logical names?